anthos-bm-gcp-terraform/modules/init/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "project_id" {}
+
+variable "zone" {}
+
+variable "hostname" {}
+
```

Comment:
Good to have types assigned here. I'm also thinking if we should have few defaults in places like ip addresses 🤔

Reply:
Yes I will add `types`, `defaults` and `descriptions`. I have added `region`, `zone`, `hostname`, ` clustername` defaults in some places.

Only place we can have default IPs is for the `vxlan` IPs. But I am autogenerating it based on the number of hosts per layer depending on the names for the `instance_names` variable. We would have to ask the user to fill in two places if we allowed for IPs as variable too. One thing we can do is take an `ip` subnet or prefix like `10.20.0.0/24` for the `vxlan`

```
variable "instance_names" {
  type = map(any)
  default = {
    "admin" : ["ws"]
    "controlplane" : ["cp1", "cp2", "cp3"]
    "worker" : ["w1", "w2"]
  }
}
```

Reply:
Added descriptions and type info in https://github.com/GoogleCloudPlatform/anthos-samples/pull/11/commits/bc08ad03dce9d97fa82d634ceebf4f62906b401f